features/multiple_apps.feature

```diff
@@ -20,7 +20,7 @@ Feature: Multiple Applications Test
    #Confirm 2 apps on Login Screen
     Then I go back
     Then I see the app selector button
-    Then I press "Test: List Searching"
+    Then I press view with id "app_selection_spinner"

```

Comment:
Will pull, but checking on whether there's a way for us to define a step for this which will make it easier to coordinate in case the value changes.